Project.toml

```diff
@@ -3,6 +3,9 @@ uuid = "4803f6b2-022a-4c1b-a771-522a3413ec86"
 version = "0.3.1"
 
 [deps]
+IntervalArithmetic = "d1acc4aa-44c8-5952-acd4-ba5d80a2a253"
+IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
+LazySets = "b4f0291d-fe17-52bc-9479-3d1a343d9043"
```

Comment:
@mforets I suggest that we modify the code to use `Requires`.

Reply:
Sounds good. I can address that in a follow-up PR.